docs/faq.md

```diff
@@ -0,0 +1,89 @@
+# Frequently Asked Questions
+
+## What is Jupytext?
```

Comment:
I'd emphasize the two-way integration more prominently. The first sentence of the paragraph makes it sound redundant with nbconvert, I think the exceptional focus of jupytext is *two-way* conversion, rather than just one-way conversion. Something like "Jupytext is a Python package that provides two-way conversion between Jupyter Notebooks and several other text-based formats."

Reply:
Yes and no to the two way conversion? I've started looking at how I could go 'ipynb free', authoring either in just markdown or py files ([md only demo / works in MyBinder](https://github.com/ouseful-template-repos/jupytext-md/tree/no-ipynb)) perhaps with a next step later on of saving rendered notebooks as HTML or PDF (an output format) rather than ipynb.

Under this way of working, Jupytext is used to make python/markdown files editable/ executable as notebooks, but not saveable as notebooks, which makes it a one way process?

Reply:
Well, I see your point - you don't convert to an ipynb file any more. When you render the notebook, it's only the text to notebook conversion that is involved. Still I think that putting the emphasis on the two-way conversion is good: people know that the way back to the notebook is the least common part.